miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/UserInfoBridge.kt

```diff
@@ -92,28 +96,43 @@ internal class UserInfoBridge {
 
     private fun onHasAccessTokenPermission(callbackObj: CallbackObj) {
         val tokenPermission = parseAccessTokenPermission(callbackObj)
+        if (doesMatchManifest(tokenPermission)) {
 
-        val successCallback = { accessToken: TokenData ->
-            bridgeExecutor.postValue(callbackObj.id, Gson().toJson(accessToken))
-        }
-        val errorCallback = { message: String ->
-            bridgeExecutor.postError(callbackObj.id, "$ERR_GET_ACCESS_TOKEN $message")
-        }
+            val successCallback = { accessToken: TokenData ->
+                accessToken.accessTokenScope = tokenPermission
+                bridgeExecutor.postValue(callbackObj.id, Gson().toJson(accessToken))
+            }
+            val errorCallback = { message: String ->
+                bridgeExecutor.postError(callbackObj.id, "$ERR_GET_ACCESS_TOKEN $message")
+            }
 
-        try {
-            userInfoBridgeDispatcher.getAccessToken(miniAppId, tokenPermission, successCallback, errorCallback)
-        } catch (e: MiniAppSdkException) {
-            if (e.message == "The `UserInfoBridgeDispatcher.getAccessToken` $NO_IMPL")
-                userInfoBridgeDispatcher.getAccessToken(miniAppId, successCallback, errorCallback)
-            else
-                throw e
+            try {
+                userInfoBridgeDispatcher.getAccessToken(miniAppId, tokenPermission, successCallback, errorCallback)
+            } catch (e: MiniAppSdkException) {
+                if (e.message == "The `UserInfoBridgeDispatcher.getAccessToken` $NO_IMPL")
+                    userInfoBridgeDispatcher.getAccessToken(miniAppId, successCallback, errorCallback)
+                else
+                    throw e
+            }
+        } else
+            bridgeExecutor.postError(callbackObj.id, "$ERR_GET_ACCESS_TOKEN $ERR_ACCESS_TOKEN_NOT_MATCH_MANIFEST")
+    }
+
+    private fun doesMatchManifest(tokenScope: AccessTokenScope): Boolean {
```

Comment:
Do we have any plan to match the entire manifest here in future? if no, then maybe we can rename it to more specific e.g. `doesAccessTokenMatch` or something like that..